certificates/sdrs-staging.pem

```diff
@@ -0,0 +1,38 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
might be better to download the certificate in the Dockerfile - to discuss with TA